src/models/modules/bricks/convbnact.py

```diff
@@ -0,0 +1,46 @@
+"""TorchOK ConvBnAct module."""
+import torch
+import torch.nn as nn
+
+
+class ConvBnAct(nn.Module):
+    """Combination of convolution, batchnorm and activation layers."""
+
+    def __init__(self,
+                 in_channels: int,
+                 out_channels: int,
+                 kernel_size: int,
+                 padding: int,
+                 stride: int = 1,
+                 bias: bool = False,
+                 act_layer: nn.Module = nn.ReLU):
```

Comment:
Make it optional. And below when it's called just check for None. Using `nn.Identity` isn't a common style